src/index.js

```diff
@@ -473,6 +476,10 @@ const initialize = async () => {
     }
 
     if (deployedContractAddress) {
+      // Deeplink
+      const erc20Decimals = '4';
```

Comment:
Since the provider won't be found (you are using a regular browser), we cannot access to the functionalities of the provider, meaning we cannot get the decimals for the ERC20 token dynamically, so the default 4 decimals is used